docs/advanced-configuration.md

```diff
@@ -191,6 +191,75 @@ for the Fargate distribution has two primary differences between regular `eks` t
     node label. The Collector's ClusterRole for `eks/fargate` will allow the `patch` verb on `nodes` resources for the default API groups to allow the cluster
     receiver's init container to add this node label for designated self monitoring.
 
+## Control Plane metrics
+
+By setting `autodetect.controlPlane=true` the helm chart will set up the otel-collector agent to collect metrics from
+the control plane.
+
+To collect control plane metrics, the helm chart has the otel-collector agent on each node use the
+[receiver creator](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/README.md)
+to instantiate control plane receivers at runtime. The receiver creator has a set of discovery rules to know
+which control plane receivers to create. The default discovery rules can vary depending on the Kubernetes distribution
+and version. If your control plane is using non-standard specs, then you can provide a custom configuration so the
+otel-collector agent can still successfully connect.
+
+The otel-collector agent relies on having pod level network access to collect metrics from the control plane pods.
+Since most cloud Kubernetes as a service distributions (aks, eks, rosa, etc) don't expose the control plane pods to the
```

Comment:
It would be good to be more explicit here if we can - either calling out what _is_ supported or calling out what is not.

Reply:
Done. I called out both.